# exceptions, how-to-use-the-try-catch-block-to-catch-exceptions